lib/condorMonitor.py

```diff
@@ -839,9 +988,17 @@ def getHash(self, hash_func):
 ############################################################
 
 
-# check that req_format_els are present in in_format_list, and if not add them
-# return a new format_list
 def complete_format_list(in_format_list, req_format_els):
+    """
```

Comment:
Append the text of the docstring after `"""`

Reply:
done